client/src/components/HomeScreen.tsx

```diff
@@ -235,6 +247,7 @@ const CreateAudit: React.FC = () => {
         organizationId: meta!.organizations[0].id,
         auditName: '',
         auditType: 'BALLOT_POLLING',
+        ballotPollingType: 'BRAVO',
```

Comment:
maybe add a test that selecting different ballot polling types sends the right request to the backend